### PR TITLE
Add EAP terms to jboss-eap.adoc based on Bob's updates

### PR DESCRIPTION
**Pull request by khosford68**
Updated jboss-eap.adoc file with additional entries based on Bob's previous pull request at https://github.com/redhat-documentation/supplementary-style-guide/pull/55.

Revised descriptions to include full sentences. Cleaned up formatting and added xrefs for links between terms. Sorted entries alphabetically.

### CODE REVIEW

File: supplementary_style_guide/glossary_terms_conventions/product_conventions/jboss-eap.adoc
Comment (mportman12) [inline supplementary_style_guide/glossary_terms_conventions/product_conventions/jboss-eap.adoc:47]: `lowercase` is one word. Ditto for all later mentions.

File: supplementary_style_guide/glossary_terms_conventions/product_conventions/jboss-eap.adoc
Comment (mportman12) [inline supplementary_style_guide/glossary_terms_conventions/product_conventions/jboss-eap.adoc:106]: In other RSSG entries, the title casing instructions are included in the main glossary entry instead of being a separate glossary entry. Recommend doing so here as well to reduce the number of entries. Ditto for all subsequent uses.

File: supplementary_style_guide/glossary_terms_conventions/product_conventions/jboss-eap.adoc
Comment (mportman12) [inline supplementary_style_guide/glossary_terms_conventions/product_conventions/jboss-eap.adoc:732]: Is there an extra period after `standalone.bat`?

File: supplementary_style_guide/glossary_terms_conventions/product_conventions/jboss-eap.adoc
Comment (mportman12) [inline supplementary_style_guide/glossary_terms_conventions/product_conventions/jboss-eap.adoc:929]: This entry is of general interest beyond this specific product. Move to the main glossary.

File: supplementary_style_guide/glossary_terms_conventions/product_conventions/jboss-eap.adoc
Comment (mportman12) [inline supplementary_style_guide/glossary_terms_conventions/product_conventions/jboss-eap.adoc:1280]: This entry is of general interest beyond this product. Move to the main glossary.

File: supplementary_style_guide/glossary_terms_conventions/product_conventions/jboss-eap.adoc
Comment (mportman12) [inline supplementary_style_guide/glossary_terms_conventions/product_conventions/jboss-eap.adoc:1303]: Repetition - get rid of the extra `Write in lower case`.

File: supplementary_style_guide/glossary_terms_conventions/product_conventions/jboss-eap.adoc
Comment (mportman12) [inline supplementary_style_guide/glossary_terms_conventions/product_conventions/jboss-eap.adoc:478]: `write the rest of the word in lower case` seems like it's implied already, right? Delete. Ditto for subsequent entries.

**Review by mportman12 — CHANGES_REQUESTED** [review]
Great job! This was a huge project, good on you for tackling it. Left some comments and suggestions.

**Comment by bergerhoffer** [comment]
Hi @khosford68 - do you have any time to take a look at addressing @mportman12's feedback?

**Comment by khosford68** [comment]
Hi @bergerhoffer. Yes I plan to do it hopefully by end of tomorrow Wednesday 28 July.  Sorry for the delay.

**Comment by bergerhoffer** [comment]
No worries @khosford68, thanks! We're just adding icons to the entries but I'm holding off on the EAP ones until this PR merges, so that we don't cause you conflicts.

File: supplementary_style_guide/glossary_terms_conventions/product_conventions/jboss-eap.adoc
Comment (khosford68) [reply supplementary_style_guide/glossary_terms_conventions/product_conventions/jboss-eap.adoc:117]: Fixed

File: supplementary_style_guide/glossary_terms_conventions/product_conventions/jboss-eap.adoc
Comment (khosford68) [reply supplementary_style_guide/glossary_terms_conventions/product_conventions/jboss-eap.adoc:94]: I've removed this entry.

File: supplementary_style_guide/glossary_terms_conventions/product_conventions/jboss-eap.adoc
Comment (khosford68) [reply supplementary_style_guide/glossary_terms_conventions/product_conventions/jboss-eap.adoc:127]: I've removed this entry and will add it to the main glossary as part of a separate pull request in due course.

File: supplementary_style_guide/glossary_terms_conventions/product_conventions/jboss-eap.adoc
Comment (khosford68) [reply supplementary_style_guide/glossary_terms_conventions/product_conventions/jboss-eap.adoc:147]: Fixed

File: supplementary_style_guide/glossary_terms_conventions/product_conventions/jboss-eap.adoc
Comment (khosford68) [reply supplementary_style_guide/glossary_terms_conventions/product_conventions/jboss-eap.adoc:346]: Fixed

File: supplementary_style_guide/glossary_terms_conventions/product_conventions/jboss-eap.adoc
Comment (khosford68) [reply supplementary_style_guide/glossary_terms_conventions/product_conventions/jboss-eap.adoc:732]: Fixed

File: supplementary_style_guide/glossary_terms_conventions/product_conventions/jboss-eap.adoc
Comment (khosford68) [reply supplementary_style_guide/glossary_terms_conventions/product_conventions/jboss-eap.adoc:929]: I've removed this entry and will add it to the main glossary as part of a separate pull request in due course.

File: supplementary_style_guide/glossary_terms_conventions/product_conventions/jboss-eap.adoc
Comment (khosford68) [reply supplementary_style_guide/glossary_terms_conventions/product_conventions/jboss-eap.adoc:1280]: I've removed this entry and will add it to the main glossary as part of a separate pull request in due course.

File: supplementary_style_guide/glossary_terms_conventions/product_conventions/jboss-eap.adoc
Comment (khosford68) [reply supplementary_style_guide/glossary_terms_conventions/product_conventions/jboss-eap.adoc:1303]: Fixed

File: supplementary_style_guide/glossary_terms_conventions/product_conventions/jboss-eap.adoc
Comment (khosford68) [reply supplementary_style_guide/glossary_terms_conventions/product_conventions/jboss-eap.adoc:478]: Fixed

File: supplementary_style_guide/glossary_terms_conventions/product_conventions/jboss-eap.adoc
Comment (khosford68) [reply supplementary_style_guide/glossary_terms_conventions/product_conventions/jboss-eap.adoc:47]: Fixed

File: supplementary_style_guide/glossary_terms_conventions/product_conventions/jboss-eap.adoc
Comment (khosford68) [reply supplementary_style_guide/glossary_terms_conventions/product_conventions/jboss-eap.adoc:106]: Reworking those entries will be a big task that might need to be handled through a separate pull request in the future.

**Comment by khosford68** [comment]
Hi @bergerhoffer. I have now committed updates based on @mportman12's feedback.

File: supplementary_style_guide/glossary_terms_conventions/product_conventions/jboss-eap.adoc
Comment (mportman12) [reply supplementary_style_guide/glossary_terms_conventions/product_conventions/jboss-eap.adoc:896]: Entry appears to still be in the EAP doc.

**Comment by mportman12** [comment]
@khosford68 Left a comment on an entry that wasn't moved over to the General list.

File: supplementary_style_guide/glossary_terms_conventions/product_conventions/jboss-eap.adoc
Comment (khosford68) [reply supplementary_style_guide/glossary_terms_conventions/product_conventions/jboss-eap.adoc:896]: Hi @mportman12. The ActiveMQ entry was removed as part of the second commit performed yesterday at https://github.com/redhat-documentation/supplementary-style-guide/pull/97/commits/62bdbb168ae66ee6fbf1ecb03faa969e79b4fd15.

Comment (mportman12) [reply supplementary_style_guide/glossary_terms_conventions/product_conventions/jboss-eap.adoc:896]: Apologies for the confusion. I meant that I thought `repo`, not `ActiveMQ`, belonged in the main glossary.

Comment (khosford68) [reply supplementary_style_guide/glossary_terms_conventions/product_conventions/jboss-eap.adoc:896]: Hi @mportman12. Ok no problem. I will remove the `repo` entry from jboss_eap.adoc.

Just to clarify. Did you also mean to instruct removal of the `ActiveMQ` entry or should I leave that in jboss_eap.adoc?

Comment (mportman12) [reply supplementary_style_guide/glossary_terms_conventions/product_conventions/jboss-eap.adoc:896]: I don't know enough about ActiveMQ to say either way. If you think it's a JBoss-specific term, I trust your judgement!

Comment (khosford68) [reply supplementary_style_guide/glossary_terms_conventions/product_conventions/jboss-eap.adoc]: Hi @mportman12. I've just pushed a third commit that removes the `repo` entry and reinstates the `ActiveMQ` entry.

**Review by mportman12 — APPROVED** [review]
LGTM

File: supplementary_style_guide/glossary_terms_conventions/product_conventions/jboss-eap.adoc
Comment (bergerhoffer) [inline supplementary_style_guide/glossary_terms_conventions/product_conventions/jboss-eap.adoc:1173]: Should the undertow entries also list "WebHTTP - Undertow" in the "See also" field?

File: supplementary_style_guide/glossary_terms_conventions/product_conventions/jboss-eap.adoc
Comment (bergerhoffer) [inline supplementary_style_guide/glossary_terms_conventions/product_conventions/jboss-eap.adoc:1237]: "Microsoft Windows Server" is listed as an incorrect form, but is used in the description above. If it's incorrect, should it not be included in the description? Or maybe you could rewrite to something like "Windows Server product by Microsoft"?

File: supplementary_style_guide/glossary_terms_conventions/product_conventions/jboss-eap.adoc
Comment (bergerhoffer) [inline supplementary_style_guide/glossary_terms_conventions/product_conventions/jboss-eap.adoc:190]: Okay, so I commented earlier about the whole "Subsystem" maybe not needing to be capitalized, but I did actually like it in the term. Because all of these subsystem entries look like they're just regular words that can be used. For some reason, the first few entries had subsystem after them, but then that stops.

I'd recommend adding "subsystem" after each of the subsystem entries (both for the normal and the title/heading entries), so that it's clearer that it's a subsystem. I assume that "subsystem" is pretty much required afterward anyways.

**Review by bergerhoffer — COMMENTED** [review]
A few thoughts if you can take a look.

Thanks again for working on all of these entries!!

File: supplementary_style_guide/glossary_terms_conventions/product_conventions/jboss-eap.adoc
Comment (khosford68) [reply supplementary_style_guide/glossary_terms_conventions/product_conventions/jboss-eap.adoc]: Fixed in the latest commit

File: supplementary_style_guide/glossary_terms_conventions/product_conventions/jboss-eap.adoc
Comment (khosford68) [reply supplementary_style_guide/glossary_terms_conventions/product_conventions/jboss-eap.adoc]: Fixed

File: supplementary_style_guide/glossary_terms_conventions/product_conventions/jboss-eap.adoc
Comment (khosford68) [reply supplementary_style_guide/glossary_terms_conventions/product_conventions/jboss-eap.adoc]: Fixed

File: supplementary_style_guide/glossary_terms_conventions/product_conventions/jboss-eap.adoc
Comment (khosford68) [reply supplementary_style_guide/glossary_terms_conventions/product_conventions/jboss-eap.adoc]: Fixed

File: supplementary_style_guide/glossary_terms_conventions/product_conventions/jboss-eap.adoc
Comment (khosford68) [reply supplementary_style_guide/glossary_terms_conventions/product_conventions/jboss-eap.adoc]: This is now fixed in the latest commit where I have amalgamated the "titles and headings" entry for each subsystem into its corresponding main glossary term.

File: supplementary_style_guide/glossary_terms_conventions/product_conventions/jboss-eap.adoc
Comment (khosford68) [reply supplementary_style_guide/glossary_terms_conventions/product_conventions/jboss-eap.adoc]: This is now fixed with the amalgamation of dual entries for subsystem terms.

File: supplementary_style_guide/glossary_terms_conventions/product_conventions/jboss-eap.adoc
Comment (khosford68) [reply supplementary_style_guide/glossary_terms_conventions/product_conventions/jboss-eap.adoc]: Fixed

File: supplementary_style_guide/glossary_terms_conventions/product_conventions/jboss-eap.adoc
Comment (khosford68) [reply supplementary_style_guide/glossary_terms_conventions/product_conventions/jboss-eap.adoc]: Fixed

File: supplementary_style_guide/glossary_terms_conventions/product_conventions/jboss-eap.adoc
Comment (khosford68) [reply supplementary_style_guide/glossary_terms_conventions/product_conventions/jboss-eap.adoc:809]: Fixed

File: supplementary_style_guide/glossary_terms_conventions/product_conventions/jboss-eap.adoc
Comment (khosford68) [reply supplementary_style_guide/glossary_terms_conventions/product_conventions/jboss-eap.adoc]: Fixed

**Comment by khosford68** [comment]
Hi @bergerhoffer. I have now committed updates based on your feedback. I have also implemented @mportman12's previous suggestion to amalgamate the dual entries for each subystem into just one entry. This will help to reduce the overall number of entries. Hopefully, it should also help to avoid any confusion that might otherwise arise from having more than one entry for each subsystem.

File: supplementary_style_guide/glossary_terms_conventions/product_conventions/jboss-eap.adoc
Comment (bergerhoffer) [inline supplementary_style_guide/glossary_terms_conventions/product_conventions/jboss-eap.adoc:682]: This xref can be removed (since it just refers back to the same entry

**Review by bergerhoffer — COMMENTED** [review]
@khosford68 Thank you so much for the updates! I think this is much clearer now with the guidance for the subsystems all contained in one entry.

I see just one xref that needs to be removed if you don't mind. Then after that I'll approve it. Thanks again!

File: supplementary_style_guide/glossary_terms_conventions/product_conventions/jboss-eap.adoc
Comment (khosford68) [reply supplementary_style_guide/glossary_terms_conventions/product_conventions/jboss-eap.adoc]: Hi @bergerhoffer. I have removed that xref now in the latest commit.

**Review by bergerhoffer — APPROVED** [review]
LGTM, thanks!

**Pull request merged**
